tensornetwork/tn_keras/test_layer.py

```diff
@@ -25,7 +25,7 @@ def dummy_data(request):
 
 @pytest.fixture(params=[
     'DenseDecomp', 'DenseMPO', 'DenseCondenser', 'DenseExpander',
-    'DenseEntangler'
+    'DenseEntangler', 'DenseEntanglerOut'
```

Comment:
What is this?

Reply:
Thats an entangler model with a different output size to input size. Meant to change that to a more informative name.